docsrc/user_guide/dynamo_export.rst

```diff
@@ -0,0 +1,72 @@
+.. _dynamo_export:
+
+Torch-TensorRT (Dynamo) Backend
+========================
+This guide presents Torch-TensorRT dynamo backend which compiles Pytorch programs 
+into TensorRT engines through torch dynamo. Pytorch 2.1 introduced ``torch.export`` APIs which 
+can export graphs from Pytorch programs using torch dynamo. Torch-TensorRT dynamo 
+backend compiles these exported graphs and optimizes them using TensorRT. Here's a simple 
+usage of the dynamo backend  
+
+.. code-block:: python
+
+    import torch
+    import torch_tensorrt
+
+    model = MyModel().eval().cuda()
+    inputs = torch.randn((1, 3, 224, 224), dtype=torch.float32).cuda()
+    exp_program = torch.export(model, inputs)
```

Comment:
Switch to `torch.export.export` for `2.1` ([reference](https://pytorch.org/docs/stable/export.html))

Reply:
Additionallty, `inputs` should be a tuple of arguments to agree with the [API recommendation](https://github.com/pytorch/pytorch/blob/192477b5ba991b5e160d7264230945b49695c0e0/torch/export/__init__.py#L332)